src/content/docs/bot-protection/quick-start/nextjs.mdx

```diff
@@ -152,8 +152,18 @@ dashboard](https://app.arcjet.com).
   <LinkCard
     title="Bot protection reference"
     href="/bot-protection/reference/nextjs"
-    description="Full details about bot protection."
+    description="Full details about rate limiting."
   ></LinkCard>
+  <LinkCard
+    title="SDK reference"
+    href="/reference/nextjs"
+    description="Write tests for your rules."
```

Comment:
```suggestion
    description="Next.js SDK reference guide."
```
I missed this one, sorry!

Reply:
There's a lot and they all blur! We'll revisit this when we do the rearchitecture of the sections.